dao/src/main/kotlin/utils/Extensions.kt

```diff
@@ -37,6 +41,22 @@ import org.jetbrains.exposed.sql.SortOrder
  */
 fun Instant.toDatabasePrecision() = minus(nanosecondsOfSecond, DateTimeUnit.NANOSECOND)
 
+/**
+ * Run the provided [query] with the given [parameters] to create a [ListQueryResult]. The entities are mapped to the
+ * corresponding model objects using the provided [entityMapper].
+ */
+internal fun <T : LongEntity, E> SortableEntityClass<T>.listQuery(
```

Comment:
Nit: I find the names of the type parameters a bit confusing: `T` is used for an entity, and `E` for a model object. Would it be better to name them `E` for the entity and `M` for the model object?